src/tiny_websockets/network/teensy41/teensy41_tcp_client.hpp

```diff
@@ -59,8 +59,12 @@ namespace websockets { namespace network {
       WSString line = "";
 
       int ch = -1;
+
+      uint64_t millisBeforeReadingHeaders = millis();
       while( ch != '\n' && available()) {
+        if ( millis() - _timer > readLineTimeout) return "";
```

Comment:
I assume you need to remove this line?

Reply:
My apologies I pushed the wrong commit, this was supposed to be removed